tests/unit/evaluation/SplittingStrategy_unittest.cc

```diff
@@ -296,3 +296,90 @@ TEST(SplittingStrategy,LOO)
 		SG_UNREF(splitting);
 	}
 }
+
+TEST(SplittingStrategy, timeseries_subset_linear_splits)
+{
+	index_t num_labels, num_subsets, min_subset_size, base_size;
+	index_t runs = 10;
+
+	while (runs-- > 0)
```

Comment:
please use a for loop
`for (auto i=0; i<runs; i++)`
or even better
`(auto i : Range(3, 10))`, see `base/range.h`